### DISCUSSION TITLE
Mask 24 hour hh:mm format

### DISCUSSION THREAD

**Discussion by SossenSystems** [beholdr/maska#213] [Q&A]
Hey community, I have a question, how would you best use the 24 hour format for times with `data-maska="##:##"` so that it works correctly?
I don't know exactly how to handle `data-maska-tokens=""` and `data-maska-eager`
Regex: `/^(0[0-9]|1[0-9]|2[0-3]):[0-5][0-9]$/`

```HTML
<input v-maska data-maska="##:##" />
```

**Comment by beholdr** [comment]
You could try to use different custom tokens for different time components, something like `Hh:M#`. Or use standard tokens with hooks to post process the result.
